### ISSUE TITLE
Suggest small change in documentation

### ISSUE THREAD

**Issue by AllenDowney** [ipums/ipumspy#77]
Thanks for this package. I am finding it super useful.

I have one small suggestion on this page: https://ipumspy.readthedocs.io/en/latest/variables.html

In this example:

```
from ipumspy import readers

# read ddi and data
ddi_codebook = readers.read_ipums_ddi([ddi xml file path])
ipums_df = readers.read_microdata(ddi_codebook, [data file path])
```

The square brackets around `[ddi xml file path]` and `[data file path]` make it seem like these functions expect a list. I'd suggest:

```
ddi_xml_file_path = ...
data_file_path = ...
ddi_codebook = readers.read_ipums_ddi(ddi_xml_file_path)
ipums_df = readers.read_microdata(ddi_codebook, data_file_path)
```

**Comment by AllenDowney** [comment]
I have a question that's not clear to me from reading the documentation here:

https://ipumspy.readthedocs.io/en/latest/reference/generated/methods/ipumspy.api.IpumsApiClient.download_extract.html?highlight=download_extract

Suppose I have made an extract but I no longer have the `Extract` object that was created when I made the request. I see the extract I want in the list, so I have the number. Am I right that I could pass this number as an `extract` parameter to `download_extract`? But in that case I would also have to provide a `collection` parameter? If that's right:

* Do I pass the number as integer or string?
* Where would I get the name of the collection from?

Thanks!
